mods/ra/maps/desert-shellmap/desert-shellmap.lua

```diff
@@ -176,6 +176,10 @@ WorldLoaded = function()
 	soviets = Player.GetPlayer("Soviets")
 	viewportOrigin = Camera.Position
 
+	Trigger.AfterDelay(DateTime.Seconds(10), function()
+		MissileSilo.SendNuke(Actor303.Location)
```

Comment:
```
Fatal Lua Error: [string "BindingSupport.lua"]:30: Actor 'mslo' does not define a property 'SendNuke'
  at OpenRA.Scripting.ScriptContext.FatalError (System.String message) [0x00000] in <4dbc9595f4ec4242bf7648a430506bc4>:0 
  at OpenRA.Mods.Common.Scripting.TriggerGlobal+<AfterDelay>c__AnonStorey0.<>m__0 () [0x00000] in <8519c22846ec4f12b51f5d7177d1eefa>:0 
  at OpenRA.Effects.DelayedAction.<Tick>m__0 (OpenRA.World w) [0x00000] in <4dbc9595f4ec4242bf7648a430506bc4>:0 
  at OpenRA.World.Tick () [0x00000] in <4dbc9595f4ec4242bf7648a430506bc4>:0 
  at OpenRA.Game.InnerLogicTick (OpenRA.Network.OrderManager orderManager) [0x00000] in <4dbc9595f4ec4242bf7648a430506bc4>:0 
  at OpenRA.Game.LogicTick () [0x00000] in <4dbc9595f4ec4242bf7648a430506bc4>:0 
  at OpenRA.Game.Loop () [0x00000] in <4dbc9595f4ec4242bf7648a430506bc4>:0 
  at OpenRA.Game.Run () [0x00000] in <4dbc9595f4ec4242bf7648a430506bc4>:0 
  at OpenRA.Game.InitializeAndRun (System.String[] args) [0x00000] in <4dbc9595f4ec4242bf7648a430506bc4>:0 
  at OpenRA.Program.Main (System.String[] args) [0x00000] in <4dbc9595f4ec4242bf7648a430506bc4>:0
```

Reply:
Changing it to `ActivateNukePower` fixes it.

Reply:
I renamed the properties as pchote wanted but forgot to update testcase. Shouldn't be much problem as it is just testcase. Can you test by manually updating it?